scripts/scripts.js

```diff
@@ -132,15 +133,228 @@ export function decorateMain(main) {
   decorateQuickLinks(main);
 }
 
+function initWebSDK(path) {
+  return new Promise((resolve) => {
+    import(path)
+      .then(() => {
+      })
+      .then(() => {
+        resolve();
+      })
+      .catch(() => {
+      });
+  });
+}
```

Comment:
This should be enough:
```suggestion
function initWebSDK(path) {
  return import(path);
}
```

Do you only use alloy for Target? if not, I'd also do the configure call here, otherwise it's fine as you have it below behind the conditional

Reply:
@ramboz updated. I am using it for both analytics and target.